blocks/identity-block/utils/validate-password-pattern.js

```diff
@@ -1,9 +1,18 @@
+const SPECIAL_CHARACTERS_ALLOWED = '@$!%*?&';
```

Comment:
Should we call these allowed values out in the error message, currently it just states you need a special character - but doesn't specify which. I'll add a separate ticket for that